cparser/Elab.ml

```diff
@@ -1427,9 +1442,14 @@ let elab_expr vararg loc env a =
         | _ ->
             error "request for member '%s' in something not a structure or union" fieldname in
       (* A field of a const/volatile struct or union is itself const/volatile *)
-      { edesc = EUnop(Odot fieldname, b1);
-        etyp = add_attributes_type (List.filter attr_inherited_by_members attrs)
-                                   (type_of_member env fld) },env
+      let rec aux = function
```

Comment:
Nondescriptive function names such as `aux` are not nice.  What about `access`? or `accessor`?  or `make_access`?  or...